Library/Homebrew/dev-cmd/test-bot.rb

```diff
@@ -920,7 +920,6 @@ def sanitize_ARGV_and_ENV
     end
 
     ENV["HOMEBREW_DEVELOPER"] = "1"
-    ENV["HOMEBREW_NO_AUTO_UPDATE"] = "1"
```

Comment:
Why remove this again? The reason for this addition wasn't speed issues and unless I'm missing something, this PR is only about improving the speed of the auto-update (or update in general).